app/models/api_docs/service.rb

```diff
@@ -65,6 +66,13 @@ def base_path
     specification.base_path || self[:base_path]
   end
 
+  def host_with_port
```

Comment:
It seems that is only for presentation logic stuff, so perhaps [a presenter](https://github.com/3scale/porta/tree/master/app/presenters) or [a decorator](https://github.com/3scale/porta/tree/master/app/decorators) containing this method instead of putting it in the model.

Reply:
That sounds like a rather simple change, would you kindly create a commit suggestion?

Reply:
sure 😉 I will 👍

Reply:
@josemigallas https://github.com/3scale/porta/pull/621 😄